roles/pulp-routes/tasks/main.yml

```diff
@@ -1,47 +1,34 @@
-    _pulp_plugins:
-      - { name: "{{ ansible_operator_meta.name }}-content", path: "{{ pulp_combined_settings.content_path_prefix | default('/pulp/content/') }}", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-container", path: "/pulp/container/", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-cookbook", path: "/pulp_cookbook/content/", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
-      - { name: "{{ ansible_operator_meta.name }}", path: "/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-api-v3", path: "{{ pulp_combined_settings.api_root }}api/v3/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-auth", path: "/auth/login/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-registry-v2", path: "/v2/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-registry-extensions-v2", path: "/extensions/v2/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-registry-token", path: "/token/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-python", path: "/pypi/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-ansible", path: "/pulp_ansible/galaxy/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
-      - { name: "{{ ansible_operator_meta.name }}-galaxy-ui", path: "/ui/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc", rewrite: "/static/galaxy_ng/index.html" }
+- block:
+    - set_fact:
+        _pulp_plugins:
+          - { name: "{{ ansible_operator_meta.name }}-content", path: "{{ pulp_combined_settings.content_path_prefix | default('/pulp/content/') }}", targetPort: "content-24816", serviceName: "{{ ansible_operator_meta.name }}-content-svc" }
+          - { name: "{{ ansible_operator_meta.name }}-api-v3", path: "{{ pulp_combined_settings.api_root }}api/v3/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
+          - { name: "{{ ansible_operator_meta.name }}-auth", path: "/auth/login/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
+          - { name: "{{ ansible_operator_meta.name }}", path: "/", targetPort: "api-24817", serviceName: "{{ ansible_operator_meta.name }}-api-svc" }
```

Comment:
for what reason have you decided to leave this as is and not adding the logic to the parser so the information is extracted there too?

Reply:
My bad, I was thinking about putting a comment on why I kept this "statically", but I just gave up because I thought that wouldn't be helpful.
The above configurations are defined in _nginx.conf,_ not through the snippets. So, from what I could understand, since they should be present in every type of installation (independently of the plugins installed) I thought that it would be fine to leave them this way.

Reply:
Yeah that's true that this config would be present in every installation. It's probably matter of preference whether to let the parser extract the information ( for the consistency of the approach) or keep this as is is also fine.